integration/indexed-list/cmd_indexed_list_test.go

```diff
@@ -0,0 +1,110 @@
+//go:build !relayer
```

Comment:
the `other components` test use underscore as space. Maybe we should define a pattern to use the same for both